truss/cli/train.py

```diff
@@ -16,7 +22,7 @@ def get_args_for_stop(
 ) -> Tuple[str, str]:
     if not project_id or not job_id:
         # get all running jobs
-        job = _get_running_job(console, remote_provider, project_id, job_id)
+        job = _get_active_job(console, remote_provider, project_id, job_id)
```

Comment:
Not introduced here, but why do we need the explicit `cast` below? And since `training_project` is now a nested field, do we need to fix the accesses to `training_project_id` so we can deprecate in the API?